http4s-blaze-server/src/main/scala/com/avast/server/toolkit/http4s/Http4sBlazeServerConfig.scala

```diff
@@ -0,0 +1,31 @@
+package com.avast.server.toolkit.http4s
+
+import java.util.concurrent.TimeUnit
+
+import com.avast.server.toolkit.http4s.Http4sBlazeServerConfig.SocketOptions
+import org.http4s.blaze.channel
+import org.http4s.server.defaults
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class Http4sBlazeServerConfig(
+  listenAddress: String,
+  listenPort: Int,
```

Comment:
I would welcome the defaults back. I like the "symmetry" between default values and possible config values.

Reply:
I don't understand but as I said before I feel like these two values actually do not deserve a default because none is general enough for everybody.

Reply:
However I have added a factory method in case someone needs it - that's a good compromise.